test/smoke/testApps/TraceLog4j1_2/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/javaee" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" id="SimpleCalculator_1" version="3.0">
+  <display-name>SimpleTelemetry</display-name>
+  <filter>
```

Comment:
You can remove the `<filter>` and `<filter-mapping>` tags here (lines 4 -- 11). Since we have other tests for the filter, and the filter isn't needed for logging.

It doesn't hurt, though.

Reply:
Thanks for your comments, I will remove those useless code in next PR.